testpilot/frontend/static-src/app/views/header-view.js

```diff
@@ -74,6 +74,11 @@ export default BaseView.extend({
       credentials: 'same-origin',
       headers: { 'X-CSRFTOKEN': app.me.csrfToken }
     }).then(() => {
+      app.sendToGA('event', {
```

Comment:
Hmm, this is already added in [`settings-menu-view.js`](https://github.com/mozilla/testpilot/blob/c4c41e893120132639347e41620fc84d054670f4/testpilot/frontend/static-src/app/views/settings-menu-view.js#L59), and seems to be working well. Not sure what the difference is between these files.